### ISSUE TITLE
Infinite redirection loop at graphql.org

### ISSUE THREAD

**Issue by ZhangYiJiang** [graphql/graphql.github.io#516]
The page seems to be redirecting in an infinite loop between the HTTP and HTTPS version 

```
$ wget http://graphql.org/index.html 
--2018-05-27 14:29:13--  http://graphql.org/index.html
Resolving graphql.org (graphql.org)... 54.192.7.25, 2600:9000:2008:da00:5:5ca6:5c40:93a1
Connecting to graphql.org (graphql.org)|54.192.7.25|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://graphql.org/index.html [following]
--2018-05-27 14:29:13--  https://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: http://graphql.org/index.html [following]
--2018-05-27 14:29:15--  http://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://graphql.org/index.html [following]
--2018-05-27 14:29:16--  https://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: http://graphql.org/index.html [following]
--2018-05-27 14:29:17--  http://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://graphql.org/index.html [following]
--2018-05-27 14:29:18--  https://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: http://graphql.org/index.html [following]
--2018-05-27 14:29:20--  http://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://graphql.org/index.html [following]
--2018-05-27 14:29:21--  https://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: http://graphql.org/index.html [following]
--2018-05-27 14:29:22--  http://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://graphql.org/index.html [following]
--2018-05-27 14:29:23--  https://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: http://graphql.org/index.html [following]
--2018-05-27 14:29:25--  http://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:80... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://graphql.org/index.html [following]
--2018-05-27 14:29:25--  https://graphql.org/index.html
Connecting to graphql.org (graphql.org)|54.192.7.25|:443... connected.
HTTP request sent, awaiting response... 301 Moved Permanently
Location: http://graphql.org/index.html [following]
--2018-05-27 14:29:27--  http://graphql.org/index.html
```

**Comment by leonardfactory** [comment]
Probably related to https://github.com/graphql/graphql.github.io/issues/514

**Comment by SLVER5111q** [comment]
SLVER

**Comment by CorwinCZ** [comment]
Issue is still on, cannot acces the GprahQL documentation. Is there some work-around?

**Comment by IvanGoncharov** [comment]
> Issue is still on, cannot acces the GprahQL documentation. Is there some work-around?

I created temporary mirror here: https://ivangoncharov.github.io/

**Comment by IvanGoncharov** [comment]
I cloned master branch here: https://github.com/IvanGoncharov/IvanGoncharov.github.io
and everything is working after I removed CNAME.
I suspect that in order to fix #514 Facebook engineers added some kind of HTTPS proxy in front of GitHub pages.
But because requests are forwarded to `graphql.github.io` GH pages redirect them based on CNAME (which is `graphql.org`).
I suspect that CNAME was manually removed from `master` but #535 triggered deploy that added CNAME back and this cause this issue.

@stubailo or anyone else with commit rights Can you please remove this file: https://github.com/graphql/graphql.github.io/blob/master/CNAME

**Comment by IvanGoncharov** [comment]
@leebyron `graphql.org` is down can you please take a look. I suspect that you just need to remove https://github.com/graphql/graphql.github.io/blob/master/CNAME

**Comment by bobvanluijt** [comment]
We have the same problem, can't access the website either...

**Comment by leebyron** [comment]
I’m looking into this with Joel’s help. 

In the meantime https://graphql.github.io/ is working correctly

**Comment by JoelMarcey** [comment]
This should be fixed now. We had to invalidate a cache.

Basically on the last commit. A CNAME file got created when we didn’t need one. So that caused the loop. I believe Lee is looking into modifying the deploy script not to include CNAME on commits.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by leebyron** [comment]
Fixed in #540
